docs/data/material/migration/migrating-from-deprecated-apis/migrating-from-deprecated-apis.md

```diff
@@ -146,6 +146,190 @@ The Avatar's `imgProps` was deprecated in favor of `slotProps.img`:
  />;
 ```
 
+## Chip
+
+Use the [codemod](https://github.com/mui/material-ui/tree/HEAD/packages/mui-codemod#chip-classes) below to migrate the code as described in the following sections:
+
+```bash
+npx @mui/codemod@latest deprecations/chip-classes <path>
+```
+
+### Composed CSS classes
+
```

Comment:
hey @DiegoAndai can you suggest description here similar to [button classes description](https://github.com/mui/material-ui/pull/40675/files#diff-a5a0a00a518352d06d4ddc3013a1da70dcae1423ee73e024aae5649982f84e7eR159). Since multiple combinations of classes are deprecated, i'm not able to come up with description which address all types of combinations similar to `Button`, `Alert`

Reply:
I would do a list:

```md
The CSS classes that composed the following props were deprecated:

- `variant` | `clickable` | `deletable` and `color`
- `avatar` and `color` | `size`
- `icon` and `color` | `size`
- `deleteIcon` and `color` | `size`

Here's how to migrate:
```

What do you think?

Reply:
yeah i guess this is the best we can do given multiple combinations. `label` and `size` was missing in this list, i added in latest commit.